proto/stream_plan.proto

```diff
@@ -256,7 +256,8 @@ message FilterNode {
 message CdcFilterNode {
   expr.ExprNode search_condition = 1;
   uint32 upstream_source_id = 2;
-  repeated int32 upstream_column_ids = 3;
+  reserved "upstream_column_ids";
+  reserved 3;
```

Comment:
Since it was never used (so there was never a message persisted with #3), can we directly remove the field?

Reply:
Sounds good! Can you do it for me?